test/FastExpressionCompiler.LightExpression.UnitTests/FastExpressionCompiler.LightExpression.UnitTests.csproj

```diff
@@ -7,8 +7,15 @@
 
     <ItemGroup>
         <None Remove="*.ncrunchproject" />
-        <Compile Include="..\FastExpressionCompiler.UnitTests\AssignTests.cs;..\FastExpressionCompiler.UnitTests\BlockTests.cs;..\FastExpressionCompiler.UnitTests\CoalesceTests.cs;..\FastExpressionCompiler.UnitTests\ConditionalOperatorsTests.cs"
-                 Exclude="..\FastExpressionCompiler.UnitTests\ArithmeticOperationsTests.cs" />
+        <Compile Include="..\FastExpressionCompiler.UnitTests\**\*.cs" Exclude="..\FastExpressionCompiler.UnitTests\obj\**\*.*" />
```

Comment:
This is exactly what I wanted to do, so for the new tests to appear in both projects.